src/main/java/edu/kis/vh/nursery/list/Node.java

```diff
@@ -1,12 +1,41 @@
 package edu.kis.vh.nursery.list;
 
+/**
+ * Represents a node in a doubly linked list structure used by IntLinkedList.
+ * Stores an integer value and references to the previous and next nodes.
+ *
+ * @author
+ */
 public class Node {
 
-    public int value;
-    public Node prev, next;
+    /**
+     * The integer value stored in this node.
+     */
+    final int value;
 
+    /**
+     * Reference to the previous and next node in the list.
+     */
+    Node previous, next;
```

Comment:
should be private
Lack of encapsulation in Node class (Task 11 in "accessory i hermetyzacja")